meta-test-family.spec

```diff
@@ -19,14 +19,28 @@ Requires:       python2-avocado
 Requires:       python2-avocado-plugins-output-html
 Requires:       python-netifaces
```

Comment:
I think the other runtime requires should also be handled by the python2- subpkage. Haven't check the executables though.